Datasets/LabeledExample.swift

```diff
@@ -34,3 +34,13 @@ public struct LabeledExample: TensorGroup, KeyPathIterable, Collatable {
         data = Tensor<Float>(handle: TensorHandle<Float>(handle: _handles[dataIndex]))
     }
 }
+
+public struct TensorPair<S1: TensorFlowScalar, S2: TensorFlowScalar>: Collatable, KeyPathIterable {
+    public var input: Tensor<S1>
+    public var target: Tensor<S2>
```

Comment:
Does `target` represent an "expected output", i.e. a label? How about `label` instead?
```suggestion
    public var label: Tensor<S2>
```

Reply:
Actually, `second` please.

Reply:
Done and updated the notebooks accordingly.